recipe/migrations/libgrpc163_libprotobuf5261.yaml

```diff
@@ -1,11 +0,0 @@
-__migrator:
-  build_number: 1
-  commit_message: Rebuild for libgrp 1.63 & libprotobuf 5.26.1
-  kind: version
-  paused: true
```

Comment:
see here; it was never unpaused, because we never managed to build `protobuf` for 5.26